cns/restserver/util.go

```diff
@@ -112,9 +113,17 @@ func (service *HTTPRestService) saveNetworkContainerGoalState(req cns.CreateNetw
 	existingNCStatus, ok := service.state.ContainerStatus[req.NetworkContainerid]
 	var hostVersion string
 	var existingSecondaryIPConfigs map[string]cns.SecondaryIPConfig //uuid is key
+	var existingNCVersion string
 	if ok {
 		hostVersion = existingNCStatus.HostVersion
```

Comment:
What is HostVersion? Are you sure that is NCVersion in NMAgent?

Reply:
Looks like it is

Reply:
It it.